### PR TITLE
rename pick to split, version with both tails

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

**Comment by eparejatobes** [comment]
I'm leaving `pick` and `pickS`; it's certainly useful to have a version which doesn't accumulate the left side. The name is awful though :)

**Comment by eparejatobes** [comment]
LGTM

File: src/main/scala/cosas/klists/syntax.scala [hunk 1]
```diff
@@ -32,6 +32,26 @@ case object syntax {
...
+      def splitS[
+        E >: X <: L#Bound,
+        OL <: AnyKList,
+        X,
+        OR <: AnyKList  {type Bound = OL#Bound }
+      ](w: Witness[E])(implicit
+        splitS: AnyApp1At[SplitS[E], L] { type Y = (OL,X,OR) }
+      )
+      : (OL, X, OR) =
+        splitS(l)
```
Comment (laughedelic) [inline src/main/scala/cosas/klists/syntax.scala]: the identation here is wrong

**Comment by laughedelic** [comment]
> I'm leaving pick and pickS

I thought that we're going to rename `takeFirst`  to `pick`

**Comment by eparejatobes** [comment]
Yes, but I'm talking about something like `??[E]: DepFn1[AnyKList, (E,AnyKlist)]`

**Comment by laughedelic** [comment]
I suggest to name it `pop`

**Pull request merged**